Description: Upstream changes introduced in version 0.33~cvs20110710-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 dietlibc (0.33~cvs20110710-2) experimental; urgency=low
 .
   * Fix build for ARMv7 arches.
   * Fix FTBFS on s390.
   * Drop 0015-arm-soft-and-hard-support.diff
 .
 The person named in the Author field signed this changelog entry.
Author: Hector Oron <zumbi@debian.org>

---

Index: pkg-dietlibc/s390/setjmp.S
===================================================================
--- pkg-dietlibc.orig/s390/setjmp.S	2012-03-25 15:07:46.000000000 +0000
+++ pkg-dietlibc/s390/setjmp.S	2012-03-25 15:12:59.000000000 +0000
@@ -14,6 +14,7 @@
 
 .global _setjmp
 .type	_setjmp,@function
+_setjmp:
 	lhi	%r3,0
 .size	_setjmp,.-_setjmp
 
Index: pkg-dietlibc/s390x/setjmp.S
===================================================================
--- pkg-dietlibc.orig/s390x/setjmp.S	2012-03-25 15:07:46.000000000 +0000
+++ pkg-dietlibc/s390x/setjmp.S	2012-03-25 15:12:59.000000000 +0000
@@ -14,6 +14,7 @@
 
 .global _setjmp
 .type	_setjmp,@function
+_setjmp:
 	lhi	%r3,0
 .size	_setjmp,.-_setjmp
 
